src/lib/styles/global/colors.scss

```diff
@@ -106,6 +106,7 @@
   --neutral-blue-a45: #ccddff45;
   --neutral-blue-800-a15: #33468015;
   --neutral-blue-800-a20: #33468020;
+  --neutral-blue-1000-a65: #05070aa6;
```

Comment:
I only see this one in the Figma design. 🤔

<img width="971" alt="Screenshot 2024-11-26 at 17 28 57" src="https://github.com/user-attachments/assets/918bb883-e216-40bf-9a13-1d733fe74fab">

Reply:
We haven’t migrated to the new colors yet, but this is a good point. I’ve asked to add the missing primitive.

Reply:
You can find a new primitive here  https://www.figma.com/design/R0OMkHgUeHJFsK3woAZl4D/NNS-Library-2.0?node-id=3967-2941&node-type=canvas&t=KanbpnOxMH3PUugr-0

Reply:
I still can't find it but I trust you :) 

<img width="913" alt="Screenshot 2024-11-27 at 09 20 28" src="https://github.com/user-attachments/assets/a394520b-8dc2-47e1-bf76-abc5c6ed1701">

Reply:
The new primitive was added here
<img width="764" alt="image" src="https://github.com/user-attachments/assets/bde17ca0-3078-4e70-8ead-edd6685a3766">
But you're right, the backdrop hasn't been updated yet, 'll ping the designer.